crates/static-analysis-kernel/src/analysis/ddsa_lib/common.rs

```diff
@@ -38,3 +74,91 @@ pub fn v8_string<'s>(scope: &mut HandleScope<'s>, str: &str) -> v8::Local<'s, v8
 pub fn v8_uint<'s>(scope: &mut HandleScope<'s>, number: u32) -> v8::Local<'s, v8::Integer> {
     v8::Integer::new_from_unsigned(scope, number)
 }
+
+/// A shorthand for requiring that a v8 value be defined, otherwise an error is returned.
+pub fn expecting_var<T>(value: Option<T>, name: &str) -> Result<T, DDSAJsRuntimeError> {
+    value.ok_or_else(|| DDSAJsRuntimeError::VariableNotFound {
```

Comment:
Here, this is a good way to test the `VariableNotFound` error: we can just try a rule with an `Edit` that does not have a `start_line` or `end_line`.